ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/tissue.model.ts

```diff
@@ -41,4 +42,21 @@ export class Tissue {
       null,null,null,null,null,null,
       null,null,null,null,null, false);
   }
+
+  getAdditionalValue(colName: string): string {
```

Comment:
Consider this approach instead, which does the checks once.  The code as written now is error prone because it's got copy-pasted transformation in the if block and in the return statement.  So it's easy to change the conditional and then forget to apply the same change to the return statement.

```
const valueAtColumnKey = this.additionalValuesJson[colName];
    const valueAtCamelCaseKey = this.additionalValuesJson[DynamicValueUtilModel.convertToCamelCase(colName)];
    const valueAtLowerCaseKey = this.additionalValuesJson[colName.toLowerCase()];
    if (this.additionalValuesJson != null) {
      if (valueAtColumnKey != null) {
        return valueAtColumnKey;
      } else if (valueAtCamelCaseKey != null) {
        return valueAtCamelCaseKey;
      } else if (valueAtLowerCaseKey != null) {
        //as a last resort, try to get the value by converting the column name to lowercase
        return valueAtLowerCaseKey;
      }
    }
    return null;
```

Reply:
Thanks this is very helpful